batch/sql/estimated-current.sql

```diff
@@ -391,6 +437,7 @@ CREATE INDEX aggregated_billing_project_user_resources_by_date_v2_user ON `aggre
 DROP TABLE IF EXISTS `aggregated_batch_resources_v2`;
 CREATE TABLE IF NOT EXISTS `aggregated_batch_resources_v2` (
```

Comment:
Note to ourselves, these tables should be dropped entirely soon anyway: https://github.com/hail-is/hail/issues/13656